nightly/index.php

```diff
@@ -16,14 +16,19 @@
 
 <p> <strong>Current release series</strong>
 
-<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
-images from the stable v4.1.x series.
-<strong><font color="red">This
-is the recommended series for all users to download and
-use.</font></strong></li>
+<li><a href="v5.0.x/"><strong>v5.0.x series</strong></a>: These are
+images from the stable v5.0.x series.
+<strong><font color="red">This is the recommended series for all users
+to download and use.</font></strong></li>
+
 
 <p> <strong>Prior stable release series</strong>
 
+<li><a href="v4.1.x/"><strong>v4.1.x series</strong></a>: These are
+images from the prior stable v4.1.x series.
+<strong><font color="red">This code base is still
+supported.</font></strong></li>
+
 <li><a href="v4.0.x/"><strong>v4.0.x series</strong></a>: These are
 images from the prior stable v4.0.x series.
 <strong><font color="red">This
```

Comment:
Do you need to change the "current development" section (below) to remove the v5.0.x series?

Reply:
done